### ISSUE TITLE
Add oracle specific parameters to option enabled_cloudwatch_logs_exports in rds cluster resource

### ISSUE THREAD

**Issue by sriprasanna** [terraform-providers/terraform-provider-aws#5253]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform VersionTerraform 
v0.11.7

### Affected Resource(s)

resource "aws_db_instance"

### Terraform Configuration Files

```hcl
resource "aws_db_instance" "new-oracle-db" {
  engine                                             = "oracle-se"
  ...
  enabled_cloudwatch_logs_exports = ["alert", "audit", "listener", "trace"]
}
```

### Debug Output

https://gist.github.com/sriprasanna/e4b964086fb76bfd1befe04bb6a3a049

### Expected Behavior

The following logs should be enabled for CloudWatch exports: `alert`, `audit`, `listener` and `trace`

### Actual Behavior

Right now, the available parameters for `enabled_cloudwatch_logs_exports` are `audit`, `error`, `general` and `slowquery`. These parameters apply for MySQL but not necessarily to other Databases. Oracle for instance supports `alert`, `audit`, `listener` and `trace`, on using these parameters validation error is thrown from [this code](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_rds_cluster.go#L311).

### Steps to Reproduce

1. `terraform plan`

### References

* PR where this feature was introduced https://github.com/terraform-providers/terraform-provider-aws/pull/4875.

**Comment by bflad** [comment]
Pull request submitted: #5494

**Comment by bflad** [comment]
In version 1.32.0 of the AWS provider, releasing later this week, you will be able to declare these additional log types. 👍

**Comment by bflad** [comment]
This has been released in [version 1.32.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1320-august-16-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
